### VehicleInfo.msg

Header header
float64[] pose